website/source/docs/providers/aws/r/alb_target_group.html.markdown

```diff
@@ -31,10 +31,10 @@ resource "aws_vpc" "main" {
 
 The following arguments are supported:
 
-* `name` - (Required) The name of the target group.
-* `port` - (Required) The port on which targets receive traffic, unless overridden when registering a specific target.
-* `protocol` - (Required) The protocol to use for routing traffic to the targets.
-* `vpc_id` - (Required) The identifier of the VPC in which to create the target group.
+* `name` - (Required) The name of the target group. Change forces new resource.
```

Comment:
I'm not strictly opposed to this inclusion, but I'm not sure we make note of the `ForceNew` behavior in other resources, do we?